src/main/java/com/ning/http/client/AsyncHttpClientConfig.java

```diff
@@ -762,31 +788,13 @@ public Builder setProxyServer(ProxyServer proxyServer) {
             return this;
         }
 
-        /**
-         * Set the {@link SSLEngineFactory} for secure connection.
-         *
-         * @param sslEngineFactory the {@link SSLEngineFactory} for secure connection
-         * @return a {@link Builder}
-         */
-        public Builder setSSLEngineFactory(SSLEngineFactory sslEngineFactory) {
```

Comment:
Was unable to find usages in Jenkins plugins. I am OK with that though I would recommend to keep a method stub which would be throwing an explicit `RuntimeException` that the method is removed